chainerx_build_helper.py

```diff
@@ -56,7 +56,7 @@ def build_extension(self, ext):
             os.path.dirname(self.get_ext_fullpath(ext.name)))
         cmake_args = [
             '-DCHAINERX_BUILD_PYTHON=1',
-            '-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=' + extdir,
```

Comment:
Should we not keep `CMAKE_LIBRARY_OUTPUT_DIRECTORY` defined?

Reply:
Defining `CMAKE_LIBRARY_OUTPUT_DIRECTORY` seems a bad hack to me.
Built library should be placed to build directory until it's installed.

What I want is a C++ headers corresponding to the `libchainerx.so` so the headers should be installed with the libraries so I'm using the cmake's install feature.